softdev.bib

```diff
@@ -627,6 +627,30 @@ @inproceedings{jalili82building
     organization={ACM}
 }
 
+@article{larcheveque95optimal,
+    author = {Larchevêque, J.-M.},
+    year = {1995},
+    month = Jan,
+    pages = {1--15},
+    title = {Optimal incremental parsing},
+    volume = {17},
+    booktitle = {ACM Trans. Program. Lang. Syst.},
+    url = {http://doi.acm.org/10.1145/200994.200996},
+    doi = {10.1145/200994.200996},
+}
+
+@article{petrone95reusing,
+    author = "Petrone, Luigi",
+    editor = "Thiagarajan, P. S.",
```

Comment:
We don't include editor fields.

Reply:
Unless this is a chapter in a book?

Reply:
Yes, looks like this is a book.

Reply:
https://link.springer.com/chapter/10.1007/3-540-60692-0_44

Reply:
These look like conference proceedings to me (FSTTCS), so this should be `inproceedings`?

Reply:
Fixed.